### DISCUSSION TITLE
Unable to stop werkzeug from emitting Errors

### DISCUSSION THREAD

**Discussion by dantebarba** [getsentry/sentry-python#2402] [Q&A] [answered]
Hi

I'm currently using Sentry SDK 1.31.0 from pip with flask integration (flask 1.x) and I'm unable to stop Sentry from reporting werkzeug log events as errors. Looks like werkzeug by default emits events different than code 2xx as errors. Is there a way to configure this with Sentry SDK or werkzeug itself?

<img width="748" alt="image" src="https://github.com/getsentry/sentry-python/assets/6656541/36e1089b-b233-4f3c-9131-065e502998ba">

**Comment by dantebarba** [comment]
I ended up filtering the messages but it would be great to have a permanent solution for it.

**Comment by sentrivana** [comment, marked as answer]
Hey @dantebarba, sorry for the late reply. Based on what you write I assume it's our logging integration capturing the werkzeug logs and raising them as errors in Sentry.

The best way to go about this it to use the [`ignore_logger`](https://github.com/getsentry/sentry-python/blob/c8154be61eb473d954db2a998b4647a64065a73e/sentry_sdk/integrations/logging.py#L48-L59) function with the werkzeug logger name. This tells the logging integration to ignore that specific logger. An example how to use it from our codebase: https://github.com/getsentry/sentry-python/blob/c8154be61eb473d954db2a998b4647a64065a73e/sentry_sdk/integrations/celery.py#L118-L119
